pkg/arm/v9/resources.go

```diff
@@ -51,8 +52,8 @@ const (
 	vmssAdminUsername                             = "cloud-user"
 )
 
-func (g *simpleGenerator) vnet() *network.VirtualNetwork {
```

Comment:
Im not sure I am big fan of putting all objects under one package.
I think these could go under `https://github.com/openshift/openshift-azure/tree/master/pkg/util/azureclient` individual subpackages, replicating SDK structure.

Reply:
@mjudeikis this is not ready for review: it's a very dirty PR :) I'm looking into what we need to change in our code base to be able to upgrade to the latest version of SDK and this PR is more like notes for me.

Reply:
I mean, I'll be happy to rearrange it later, if we decide to go this way at the end.